lib/cog_api/resources/role.ex

```diff
@@ -4,5 +4,6 @@ defmodule CogApi.Resources.Role do
   defstruct [
     :id,
     :name,
+    permissions: [%CogApi.Resources.Permission{}],
```

Comment:
Can you modify the tests for both the `HTTP` client and the `Fake` that verify this behavior?